docs/references/astro/migrating-from-astro-community-sdk.mdx

```diff
@@ -38,91 +38,90 @@ With Clerk's official SDK, `clerk-js` is hotloaded by default, and the option to
 
 The following changes are required in your Astro configuration file:
 
-```diff {{ prettier: false }}
+```jsx {{ prettier: false, del: [[4, 7]], ins: [8] }}
 import { defineConfig } from "astro/config";
 import node from "@astrojs/node";
 
-- import clerk from "astro-clerk-auth/hotload";
-- import clerk from "astro-clerk-auth/integration/hotload";
-- import clerk from "astro-clerk-auth";
-- import clerk from "astro-clerk-auth/integration";
-+ import clerk from "@clerk/astro";
+import clerk from "astro-clerk-auth/hotload";
+import clerk from "astro-clerk-auth/integration/hotload";
+import clerk from "astro-clerk-auth";
+import clerk from "astro-clerk-auth/integration";
+import clerk from "@clerk/astro";
 
 export default defineConfig({
-      integrations: [clerk()],
-      adapter: node({ mode: "standalone" }),
-      output: "server",
+  integrations: [clerk()],
+  adapter: node({ mode: "standalone" }),
+  output: "server",
 });
 ```
 
 ### Prefix for environment variables
 
 The prefix for environment variables used in client-side code has been updated to align with Astro's [best practices](https://docs.astro.build/en/guides/environment-variables/). This change affects all [environment variables](https://clerk.com/docs/deployments/clerk-environment-variables) supported by Clerk.
 
-```diff {{ prettier: false }}
-- PUBLIC_ASTRO_APP_CLERK_PUBLISHABLE_KEY=
-+ PUBLIC_CLERK_PUBLISHABLE_KEY=
+```{{ prettier: false, del: [1, 4, 7], ins: [2, 5, 8] }}
```

Comment:
Going to leave this as plain text for now because the `*` is invalid and causes some inconsistency in the highlighting:

![CleanShot 2024-08-01 at 11 52 53@2x](https://github.com/user-attachments/assets/2ef40c7f-70ef-46fb-9c05-a3c9127c02f3)

Do you think there's a way to rewrite this section where the env vars only contain `_` and letters?